test/release/examples/primers/arrays.chpl

```diff
@@ -372,18 +370,26 @@ forall (i,j) in ProbSpace do
 
 writeln("Y is:\n", Y);
 
+/*
+   Our current implementation does not yet support arrays of arrays
+   where the inner array size is a function of the outer -- they must
+   all be of uniform size.  In particular, it is our intention to
+   support things like:
+
+   .. code-block:: chapel
+
+       var Triangle: [row in 1..n] [1..row] real;
+       var HierArr: [lvl in 1..n] [1..2**lvl, 1..2**lvl] real;
+
+   If such capabilities would be of use to you, please let us know and
+   we will adjust their priority accordingly.
+*/
+
 //
-// Our current implementation does not yet support arrays of arrays
-// where the inner array size is a function of the outer -- they must
-// all be of uniform size.  In particular, it is our intention to
-// support things like:
-//
-//   const Triangle: [1..row] real = [row in 1..n] [col in 1..row] row + col/10.0;
-//
-// and:
-//
-//   var HierArr: [lvl in 1..n] [1..2**lvl, 1..2**lvl] real;
-//
-// If such capabilities would be of use to you, please let us know and
-// we will adjust their priority accordingly.
+// For further information, see the domain primer
+// :ref:`domains.chpl <primers-domains>`
```

Comment:
I like calling these by their title names, e.g. "Domains Primer", rather than the file names "domains.chpl"